TypeCobol/Compiler/Sql/Model/TruncateClause.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.IO;
```

Comment:
About the file name: TRUNCATE is not a clause but a statement. Could you rename the file to `Truncate.cs` ?